lib/postprocessing/image.js

```diff
@@ -328,6 +278,56 @@ async function prepareImagePostProcess(rendition, options) {
     // return undefined if no instruction change needed
 }
 
+async function getImageMetadata(file) {
```

Comment:
I changes the order in the file so metadata-related functions are together towards the bottom, as there are good chances that `metadata` stuff will go into its own file.